include/swift/AST/Module.h

```diff
@@ -260,6 +260,11 @@ class ModuleDecl : public DeclContext, public TypeDecl {
   /// Add a file declaring a cross-import overlay.
   void addCrossImportOverlayFile(StringRef file);
 
+  /// Returns true if this module is known to not declare any cross-import
+  /// overlays. This doesn't actually load the files, so \c true may be
+  /// incorrect but \c false is never wrong.
```

Comment:
Should be `\c false may be incorrect but \c true is never wrong`.

Reply:
Drive-by suggestion: would it help to make this a 2-state enum? NoCrossImportOverlays and MayHaveCrossImportOverlays?

Reply:
I decided that's too heavyweight; instead, I'm flipping the sense of the return value and changing its name to `mightDeclareCrossImportOverlays()`, which accurately conveys the uncertainty.